packages/lib/src/core/core.ts

```diff
@@ -121,22 +123,40 @@ class Core implements ICore {
         return Promise.resolve(this);
     }
 
+    private async fetchLocaleTranslations(): Promise<Translations> {
+        try {
+            const translation = await getTranslations(
+                this.cdnContext,
+                Core.metadata.version,
+                this.options.locale,
+                this.options._translationEnvironment,
+                this.options.translations
+            );
```

Comment:
With so many arguments - you could clean up the function signature a bit by passing a single object and extracting the props in `get-translations.ts`

Reply:
I prefer to be more strict and pass each prop individually in this case.

Otherwise I'd end up passing the whole `CoreConfiguration` object to the function without any reason. Besides that, I'd have to add 'similar' piece of code in the `get-translations` function itself, to extract the values correctly